src/scanner.jl

```diff
@@ -13,6 +13,8 @@ is_s_white(c::Char) = c == yaml_1_2_s_space || c == yaml_1_2_s_tab
 # YAML 1.2 [37] ns-ascii-letter ::= [x41-x5A] | [x61-x7A] # A-Z a-z
 is_ns_ascii_letter(c::Char) = 'A' ≤ c ≤ 'Z' || 'a' ≤ c ≤ 'z'
 
+is_whitespace(::YAMLV1_1, c::Char) = c == '\0' || c == ' ' || c == '\t' || is_b_char(YAMLV1_1(), c)
```

Comment:
Counting `\0` as whitespace seems suspicious but maybe that's the best way to make sense of the existing code.

Reply:
I haven't understood this `whitespace` meaning yet but only mechanically replacing existing code.